app/models/spree/variant.rb

```diff
@@ -63,6 +63,7 @@ class Variant < ApplicationRecord
 
     validate :check_currency
     validates :price, numericality: { greater_than_or_equal_to: 0 }, presence: true
+    validates :shipping_category, presence: true
```

Comment:
This won't be necessary once https://github.com/openfoodfoundation/openfoodnetwork/pull/11297/files#diff-833d4905682446957f9fc24c1b8d02bf9e1cb594484989aa9d8c921a84aab69e is merged.

I think it's worth waiting until that is merged, then we can remove this line. 

Alternatively in the meantime, it would make it easier to later identify the unnecessary rule by updating the relationship like so:
```
belongs_to :shipping_category, class_name: 'Spree::ShippingCategory', optional: false
```

Reply:
My mentioned PR is keeping the old default for this class. So the `optional: false` is the best solution for now but I don't want that to halt this pull request.